### PR TITLE
Boot to windows permanently changes connection method to ssh

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

File: self-tests/os-boot.robot [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Suite Setup         Run Keyword
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+BOT001.001 Boot To Ubuntu Multiple Times
+    [Documentation]    This test verifies if the DUT can boot to Ubuntu multiple times in a row.
+    FOR    ${i}    IN RANGE    5
```
Comment (macpijan) [inline self-tests/os-boot.robot:34]: @JanPrusinowski Please add: `Depends On    ${TESTS_IN_UBUNTU_SUPPORT}`

File: self-tests/os-boot.robot [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        Log To Console    Iteration: ${index}
+        Power On
+        Boot System Or From Connected Disk    ubuntu
+        Login To Linux
+        Switch To Root User
+    END
+
+BOT002.001 Boot To Windows Multiple Times
+    [Documentation]    This test verifies if the DUT can boot to Windows multiple times in a row.
+    FOR    ${i}    IN RANGE    5
```
Comment (macpijan) [inline self-tests/os-boot.robot:45]: @JanPrusinowski Please add: `Depends On    ${TESTS_IN_WINDOWS_SUPPORT}`

File: self-tests/os-boot.robot [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        ${index}=    Evaluate    ${i} + 1
+        Log To Console    Iteration: ${index}
+        Power On
+        Login To Windows
+        ${out}=    Execute Command In Terminal    ls
+    END
+
+BOT003.001 Boot To Ubuntu Then Boot To Windows
+    [Documentation]    This test verifies if the DUT can boot to multiple OS one after another multiple times.
+    FOR    ${i}    IN RANGE    5
```
Comment (macpijan) [inline self-tests/os-boot.robot:55]: Please depend both on Ubuntu and Windows @JanPrusinowski

File: self-tests/os-boot.robot (hunk 1 above)
Comment (JanPrusinowski) [reply self-tests/os-boot.robot:35]: added in: 91e315ab84607c918e18a9bcaf8291a869ac14ff

File: self-tests/os-boot.robot (hunk 2 above)
Comment (JanPrusinowski) [reply self-tests/os-boot.robot:47]: added in: 91e315ab84607c918e18a9bcaf8291a869ac14ff

File: self-tests/os-boot.robot (hunk 3 above)
Comment (JanPrusinowski) [reply self-tests/os-boot.robot:59]: added in: 91e315ab84607c918e18a9bcaf8291a869ac14ff

**Pull request merged**